v4l2/videodev.py

```diff
@@ -92,9 +92,10 @@ def find_video_device(key: str, value: str) -> str:
 
         raise FileNotFoundError(f'No video device "{key}" = "{value}" found')
 
-    def get_formats(self, buf_type: v4l2.BufType):
+    def get_formats(self, buf_type: v4l2.BufType, mbus_code: v4l2.BusFormat = None):
         fmt = v4l2.uapi.v4l2_fmtdesc()
         fmt.type = buf_type.value
+        fmt.mbus_code = 0 if mbus_code is None else mbus_code
```

Comment:
Also, I'd rather write this as `mbus_code if mbus_code else 0`. I'll change that too when picking this up.